src/Model/Domain/UnsubscribeTracking.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\Domain;
+
+/**
+ * Represents a single Unsubscribe Tracking setting for a domain tracking.
+ *
+ * @author Artem Bondarenko <artem@uartema.com>
+ */
+final class UnsubscribeTracking
+{
+    private $active;
+    private $htmlFooter;
+    private $textFooter;
+
+    public static function create(array $data): self
+    {
+        $model = new self();
+        $model->active = ($data['active'] ?? null) ? 'true' : 'false';
```

Comment:
Suggesting setting `yes` or `no` so that we can keep consistency with the other two models

```suggestion
        $model->active = ($data['active'] ?? null) ? 'yes' : 'no';
```